test/test.js

```diff
@@ -38,3 +38,19 @@ x509.verify(
     assert.throws(assert.ifError.bind(null, err), /ENOENT/)
   }
 );
+
+x509.verify(
+  path.join(__dirname, 'certs/equifax.crt'),
+  path.join(__dirname, '/test.js'),
+  function(err, result) {
+    assert.notStrictEqual(err, null);
```

Comment:
Should throw an invalid format error here?

Reply:
error was caught by function verify